modules/ROOT/pages/configure-hardening.adoc

```diff
@@ -0,0 +1,23 @@
+= Configure Environment Hardening on Runtime Fabric
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+By design, Runtime Fabric automatically provides multiple aspects of environment hardening for your Mule installation. 
+For example, by leveraging Docker and Kubernetes, app isolation is achieved by default.  
+
+The following hardening checklist provides additional information about environment hardening already provided by 
+Runtime Fabric, and specifies Runtime Fabric coverage as well as areas that require additional configuration.
+
+== Hardening Checklist
+
+[%header%autowidth.spread]
+|===
+| Hardening Step | Action Required
+| Run Mule as a Non-privileged User. | RTF provides this functionality.
```

Comment:
non

Reply:
got it - thank you!

Reply:
Unsure if we've introduced the term "RTF" in our docs (some might associate it with "rich text format"). 

Ask to check in order to be consistent.

Reply:
"Runtime Fabric runs Mule applications as a non-privileged user by default.: